index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+</head>
+
+<body>
+    <h>Hello World!, we are software developers</h>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script src="js/index.js"></script>
+</body>
+
+</html>
```

Comment:
- Add new line at the end

Reply:
- [ ] Nice try. The comment is polite enough. However, you forgot to add the reason for the changes you are requesting. Adding the reason for the requested changes helps to author to understand the importance of the changes. Let's add the reason for the requested changes as requested by the previous reviewer, please.